programs/bpf_loader/src/syscalls/mem_ops.rs

```diff
@@ -550,6 +554,41 @@ impl<'a> DoubleEndedIterator for MemoryChunkIterator<'a> {
             }
         };
 
+        let region_is_account;
+
+        let mut account_index = self
+            .account_index
+            .unwrap_or_else(|| self.accounts.len().saturating_sub(1));
+        self.account_index = Some(account_index);
```

Comment:
`get_or_insert_with()`

Reply:
You're right, this is nicer.  Thanks! I'll create a PR on master